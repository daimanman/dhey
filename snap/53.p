{"C":1,"Duration":63670602334,"EndTime":"2018-08-23T14:25:34.9805083+08:00","Err":"","FileId":"4","Method":"POST","N":1,"P":{"info":"TEssssssssskkkkkk颠三倒四ST","name":"sssssDXwwM"},"Payload":"","QH":{},"Remark":"压力测试","ReqNums":1,"StartTime":"2018-08-23T14:25:34.9705152+08:00","Status":1,"TaskId":53,"Type":"FIXED","Url":"http://localhost:9188/go1","Z":""}