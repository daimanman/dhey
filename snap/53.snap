{"AvgTotal":0.0089969,"Fastest":0.0089969,"Slowest":0.0089969,"Average":0.0089969,"Rps":111.14939590303327,"AvgConn":0.0037805,"AvgDNS":0.0037805,"AvgReq":0,"AvgRes":0,"AvgDelay":0.0030188,"ConnMax":0.0037805,"ConnMin":0.0037805,"DnsMax":0.0037805,"DnsMin":0.0037805,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0,"DelayMax":0.0030188,"DelayMin":0.0030188,"Lats":[0.0089969],"ConnLats":[0.0037805],"DnsLats":[0.0037805],"ReqLats":[0],"ResLats":[0],"DelayLats":[0.0030188],"Total":8996900,"ErrorDist":{},"StatusCodeDist":{"200":1},"SizeTotal":62,"SizeReq":62,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.0089969,"Count":1,"Frequentcy":1},{"Mark":0.0089969,"Count":0,"Frequentcy":0},{"Mark":0.0089969,"Count":0,"Frequentcy":0},{"Mark":0.0089969,"Count":0,"Frequentcy":0},{"Mark":0.0089969,"Count":0,"Frequentcy":0},{"Mark":0.0089969,"Count":0,"Frequentcy":0},{"Mark":0.0089969,"Count":0,"Frequentcy":0},{"Mark":0.0089969,"Count":0,"Frequentcy":0},{"Mark":0.0089969,"Count":0,"Frequentcy":0},{"Mark":0.0089969,"Count":0,"Frequentcy":0},{"Mark":0.0089969,"Count":0,"Frequentcy":0}],"TaskId":53,"Duration":0.0089969}